framework/grid/GridView.php

```diff
@@ -463,7 +463,7 @@ public function renderTableBody()
         $rows = [];
         foreach ($models as $index => $model) {
             $key = $keys[$index];
-            if ($this->beforeRow !== null) {
+            if (is_callable($this->beforeRow)) {
```

Comment:
This make absolutely no sense, since `beforeRow` accept only `null` or `callable`.
Such change produces silent fall in case user make a mistake in configuration.